tests/test_moe/test_moe_checkpoint.py

```diff
@@ -6,57 +6,127 @@
 import pytest
 import torch
 import torch.distributed as dist
+from torch.utils.data import Dataset
 from transformers.models.llama import LlamaConfig
 
 import colossalai
 from colossalai.booster import Booster
 from colossalai.booster.plugin.moe_hybrid_parallel_plugin import MoeHybridParallelPlugin
 from colossalai.moe.manager import MOE_MANAGER
-from colossalai.testing import rerun_if_address_is_in_use, spawn
+from colossalai.testing import check_state_dict_equal, rerun_if_address_is_in_use, spawn
+from colossalai.utils import get_current_device
 
-sys.path.append(os.path.join(
-    os.path.dirname(os.path.dirname(os.path.dirname(__file__))),
-    "examples/language/openmoe",
-))
+sys.path.append(
+    os.path.join(
+        os.path.dirname(os.path.dirname(os.path.dirname(__file__))),
+        "examples/language/openmoe",
+    )
+)
 
 OpenMoeForCausalLM = importlib.import_module("model.modeling_openmoe").OpenMoeForCausalLM
 set_openmoe_args = importlib.import_module("model.modeling_openmoe").set_openmoe_args
 OpenMoeForCausalLMPolicy = importlib.import_module("model.openmoe_policy").OpenMoeForCausalLMPolicy
 
 
+class RandomDataset(Dataset):
```

Comment:
Maybe you  can use `colossalai.testing.DummyDataloader`

Reply:
done